Source/Scene/Model.js

```diff
@@ -2958,6 +2958,7 @@ function createTexture(gltfTexture, model, context) {
       pixelDatatype: texture.type,
       sampler: sampler,
       flipY: false,
+      skipColorSpaceConversion: true,
```

Comment:
I forgot to mention this earlier - can you add this option to `GltfTextureLoader` as well? That file is part of the `Model.js` rewrite that @ptrgags and @sanjeetsuhag are working on but we might as well make the fix there now. See https://github.com/CesiumGS/cesium/blob/master/Source/Scene/GltfTextureLoader.js#L272